src/main/java/core/basesyntax/strategy/operation/SupplyOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {

```

Comment:
The class `SupplyOperation` is missing a package declaration. According to the checklist, don't keep all logic in a single package. Make sure to organize your classes into appropriate packages.